Users/Install.rst

```diff
@@ -83,10 +83,77 @@ To install coala only (without any bears), you can do:
     **Ubuntu Users** - Do not use ``apt-get install coala``
     for installing coala as that is different software.
 
+Installing inside a pipenv
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pipenv is  user friendly method of managing virtual environments and
+packages.
+You can look at it as a mix of ``pip`` and ``virtualenv``. So you don't have
+to manage them seperately for your projects.
+
+First, assuming you have installed python and pip earlier from
+this guide we'll now install ``pipenv`` by running the following command.
+
+::
+
+    $ pip install pipenv
```

Comment:
do we refer to pip with `pip` or `pip3` in the docs? Because it's different for Windows and Linux users^^
If it's `pip`, then nvm ;)

Reply:
It'll not matter which command is used to install `pipenv` in either case. The environment will be initialized later on using `pipenv --three`.

Reply:
anyway please be consistent, I've just checked it and we use `pip3` everywhere :+1: